features/CCC_CFE_Integration_Test_V3/DependantNP2_Threshold-V6-2023.feature

```diff
@@ -0,0 +1,54 @@
+Feature:
+    "Certificated assessment for an applicant in domestic abuse proceedings.
+    The client has 1 child dependants and 1 adult dependants (one of which has income) and post 10th April 2023 rates applied.
+    Income from student loan, outgoings for childcare at irregular dates and capital includes a bank account.
+    The disposable income and capital is over the lower threshold therefore the result is income and capital contributions required."
+
+  Scenario: Test that the correct output is produced for the following set of data.
+    Given I am undertaking a certificated assessment
+    And A submission date of "2023-04-10"
+    And A domestic abuse case
+    And I have a dependant aged 13
+    And I have a dependant aged 25 with monthly income of 340 and capital 7999
+    And I add the following irregular_income details in the current assessment:
+      | income_type  | frequency | amount  |
+      | student_loan | annual    | 12000.00|
+    And I add multiple outgoing details including "child_care" of 200 per month, with bespoke dates: "2019-04-15" "2019-04-15" "2019-03-15"
```

Comment:
I think these dates are also wrong

Reply:
Corrected, thanks for spotting @MazOneTwoOne